src/autoscaler/db/sqldb/policy_sqldb.go

```diff
@@ -72,6 +72,29 @@ func (pdb *PolicySQLDB) GetAppIds() (map[string]bool, error) {
 	return appIds, nil
 }
 
+func (pdb *PolicySQLDB) GetAppIdsWithPolicy(policyGuid string) ([]string, error) {
```

Comment:
Rename the function to GetAppIdsWithPolicyGuid?

Reply:
Removed it, by returning the affected rows in `ReplaceAppPolicies` and `DeletePoliciesByPolicyGuid`, which also reduces the number of queries.